modules/dev-tools/docs/cli/ocular-publish.md

```diff
@@ -18,6 +18,9 @@ npx ocular-publish [mode] [npm-tag]
 
 - `beta` - bump pre-release version and publish with beta flag.
 - `prod` - bump patch version and publish.
+- `version-beta` - bump pre-release version only.
+- `version-prod` - bump patch version only.
+- `from-git`: publish from the current git tag.
```

Comment:
Wording nit: What is "publish from" intended to convey? I assume we always publish from the checked out commit. So, what does this actually do? Scrapes git to see if there are tags on the currently checked out commit, and if so picks one of those? 

```suggestion
- `from-git`: publish with the npm tag set to the git tag of the currently checked out commit. Fails if the checked out commit doesn't have a tag.
```

Reply:
Related:

https://lerna.js.org/docs/features/version-and-publish#from-package

Lerna has both `from-git` and `from-package` commands. I'm not sure why `from-git` doesnt't seem to be documented. For our purposes, probably either would be fine.